src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -276,18 +276,21 @@ private void disableSimulationItems()
          updateSimulationItemsAccordingToSimulationState();
       }
 
-      private void updateSimulationItemsAccordingToSimulationState(bool isRunning = false)
```

Comment:
this default value was not correct. because it looks like by default it's never running and since we were combining it with the state coming from the ISRunning extension method, it would alwyas default to this value which was wrong since it's updated AFTER the event are raised